### DISCUSSION TITLE
Annotation tool

### DISCUSSION THREAD

**Discussion by scampion** [jomjol/AI-on-the-edge-device#1009] [Ideas] [answered]
Hi Guys, 
Here in Belgium, we can't have direct access to the counter for electricity.
We have a kind of box with plexiglass and even for a human, it's not easy to read the number if it's dirty inside (we can't open it)

I need to train the model with that constraint and probably a specific belgium font. 
I'm working on a dedicated tool for custom annotations 
When it will be ready, I will submit a merge request to the neural-network-digital-counter-readout project if it make sense

<img width="1208" alt="image" src="https://user-images.githubusercontent.com/554155/189600608-fd53310b-7696-4456-b024-c932c73234cb.png">

**Comment by caco3** [comment, marked as answer]
Are you aware of https://github.com/haverland/collectmeterdigits?

@haverland  FYI
